tests/test_lldpSyncDaemon.py

```diff
@@ -74,17 +85,17 @@ def test_sync_roundtrip(self):
         for k in keys:
             # The test case is for LLDP neighbor information.
             # Need to filter LLDP_LOC_CHASSIS entry because the entry is removed from parsed_update after executing daemon.sync().
-            if k != 'LLDP_LOC_CHASSIS':
+            if k.decode() != 'LLDP_LOC_CHASSIS':
                 dump[k] = db.get_all(db.APPL_DB, k)
-        print(json.dumps(dump, indent=3))
+        decoded_dump = decode_dict(dump)
```

Comment:
Is this necessary? swsssdk should now return strings in both Python 2 and 3. @qiluo-msft may be able to comment further.

Reply:
@jleveque Updated in db_connector.py.